.github/workflows/ci-code.yml

```diff
@@ -73,7 +74,7 @@ jobs:
         AIIDA_WARN_v3: 1
       # Python 3.12 has a performance regression when running with code coverage
       # so run code coverage only for python 3.9.
-      run: uv run pytest -n auto --db-backend psql -m 'not nightly' tests/ ${{ matrix.python-version == '3.9' && '--cov aiida' || '' }}
+      run: uv run pytest --db-backend ${{ matrix.database-backend }} -m 'not nightly' tests/ ${{ matrix.python-version == '3.9' && '--cov aiida' || '' }}
```

Comment:
why parallel testing has been removed?
```-n auto ```

Reply:
I remove it just for test purpose to narrow down the error is caused from xdist, see my previous comment.